### ISSUE TITLE
Submit Issue to react-native about WebAssembly

### ISSUE THREAD

**Issue by martinheidegger** [consento-org/sodium-javascript#2]
There is a fix in the test setup for broken/missing WebAssembly support in the react-native for iOS system: https://github.com/consento-org/sodium-javascript/blob/6322cb10a201caa26ab70bc53a567d4db6d074e1/test.js#L7-L14 - This should be reported to react-native with the hopes to make it unnecessary to have that fix in future.

**Comment by RangerMauve** [comment]
Looking into this now

**Comment by RangerMauve** [comment]
I think there's a bug in my logic actually. 🙃

**Comment by RangerMauve** [comment]
Working on making a minimal test case to submit in the RN github issue.

I have a hypothesis that maybe WASM is actually supported but there's something weird when loading? 🤔

**Comment by RangerMauve** [comment]
https://github.com/facebook/react-native/issues/30144

**Comment by RangerMauve** [comment]
BLocked by faceboook for now, but we have a workaround

**Comment by martinheidegger** [comment]
Even if facebook reacts on this it will take ~ half-a-year until this may work. Is it possible in a react-native setup to just set `global.WebAssembly = undefined`?

**Comment by RangerMauve** [comment]
@martinheidegger Yes, I've done it in the shim.js file within the [hypercore-rn](https://github.com/consento-org/hypercore-rn/blob/main/shim.js#L8) repo.
